Buildings/Experimental/DHC/Loads/Examples/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -84,7 +90,7 @@ model BuildingTimeSeries
       each unit="W"),
     offset={0,0,0},
     columns={2,3,4},
-    smoothness=Modelica.Blocks.Types.Smoothness.MonotoneContinuousDerivative1)
+    smoothness=Modelica.Blocks.Types.Smoothness.LinearSegments)
```

Comment:
`MonotoneContinuousDerivative1` would output more smooth time series which attributes to better numerical performance.

Reply:
ok, i will modify it.